modules/oe_content_event/modules/oe_content_event_event_programme/config/install/field.field.oe_event_programme.default.event_programme_dates.yml

```
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_event_programme.event_programme_dates
+    - oe_content_event_event_programme.oe_event_programme_type.default
+  module:
+    - datetime_range
+id: oe_event_programme.default.event_programme_dates
+field_name: event_programme_dates
```

Should we have this field name?
```suggestion
field_name: oe_event_programme_dates
```